modules/stories.yml

```diff
@@ -6,7 +6,7 @@ icon: ''
 github: 'https://github.com/richardeschloss/nuxt-stories'
 website: 'https://github.com/richardeschloss/nuxt-stories'
 learn_more: ''
-category: Devtools
+category: Extensions
```

Comment:
I consider it as a Devtools or UI for this one.

Reply:
By considering "Extensions" as those ones add a change to protuction, keeping in Devtools makes sense